packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -57,11 +58,28 @@ export interface ExpansionHub extends RevHub {
     getFTDIResetControl(): Promise<boolean>;
 
     // DIO
-    setDigitalSingleOutput(dioPin: number, value?: boolean): Promise<void>;
+    setDigitalSingleOutput(digitalChannel: number, value: DigitalState): Promise<void>;
     setDigitalAllOutputs(bitPackedField: number): Promise<void>;
+
+    /**
+     * Set a digital pin as input or output.
+     * @param dioPin
+     * @param direction
+     */
     setDigitalDirection(dioPin: number, direction: DIODirection): Promise<void>;
```

Comment:
`DIODirection` should be `DioDirection` or maybe `DigitalChannelDirection`

Reply:
Changing DIODirection would be a breaking change in rhsplib. I'd prefer to create DigitalChannelDirection in expansion-hub and convert internally.

Reply:
Fine with me